src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/ActivityStatus.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.sandl.snlevents.model;
+
+@SuppressWarnings("squid:S00115")
+public enum ActivityStatus {
```

Comment:
What if at some point we'd like to track user activities? Like user creation, deletion or adding notes? You'd have to add another field to this enum which results in having 1000 fields in this enum. Should we consider changing this to an interface and then have specific enums to implement this interface?
We could have then: UserActivityStatus, HearingActivityStatus, SessionActivityStatus etc...

Reply:
good idea for future use, not necessarily needed atm (but so not mind if it;s done now)